manifest.json

```diff
@@ -1,12 +1,7 @@
 {
   "name": "Assistive pricing",
   "description": "Handles assistive pricing, and allows for users to modify labels.",
-  "ubiquity:listeners": [
-    "issues.labeled",
-    "issues.unlabeled",
-    "label.edited",
-    "issue_comment.created"
-  ],
+  "ubiquity:listeners": ["repository.created", "issues.opened", "issues.labeled", "issues.unlabeled", "label.edited", "issue_comment.created"],
```

Comment:
What is `issue_comment.created` used for?

Reply:
it's for the /allow command